tests/route/test_route_flap.py

```diff
@@ -124,10 +124,9 @@ def get_neighbor_info(duthost, dev_port, tbinfo):
     for neighbor in neighs:
         local_ip = neighs[neighbor]['local_addr']
         nbr_port = get_port_by_ip(config_facts, local_ip)
-        if 'Ethernet' in nbr_port:
-            for p_key, p_value in nbr_port_map.items():
-                if p_value == nbr_port:
-                    nbr_port = p_key
+        for p_key, p_value in nbr_port_map.items():
```

Comment:
@YatishSVC - wondering if we can use minigraph_port_alias_to_name_map to skip this loop?